velox/exec/tests/SpillTest.cpp

```diff
@@ -493,3 +493,43 @@ TEST_F(SpillTest, spillPartitionSet) {
     }
   }
 }
+
+TEST_F(SpillTest, spillPartitionSpilt) {
+  for (int iter = 0; iter < 5; ++iter) {
+    int numBatches = 100;
+    std::vector<RowVectorPtr> batches;
+    batches.reserve(numBatches);
+
+    const int numRowsPerBatch = 100;
+    setupSpillState(
+        iter % 2 ? 1 : 1'000'000'000, 1, numBatches, numRowsPerBatch);
+    const SpillPartitionId id(0, 0);
```

Comment:
I'm seeing that SpillPartitionId is a struct of 2 numbers: partitionBitOffset and partitionNumber. These fields are not const though. Any particular reason they are mutable?